Packs/JoeSecurity/Integrations/JoeSecurity/JoeSecurity.py

```diff
@@ -258,10 +258,10 @@ def analyse_sample_file_request(file_entry, should_wait, internet_access, commen
     filename = demisto.getFilePath(file_entry)['name']
     if isinstance(filename, unicode):  # py2 way of checking if a var is of type unicode
         filename = filename.encode('ascii', 'ignore')
-    filename.replace('\\', '/')
+    proper_filename = filename.replace('\\', '/')
```

Comment:
```suggestion
    fixed_filename = filename.replace('\\', '/')
```

Please add a comment explaining this addition

Reply:
There is a comment in the lines 256-257